lib/Runtime/Language/JavascriptExceptionOperators.cpp

```diff
@@ -485,10 +495,10 @@ namespace Js
             // When we start inlining functions with try, we have to track the try addresses of the inlined functions as well.
 
 #if ENABLE_NATIVE_CODEGEN
-            Assert(scriptContext->GetThreadContext()->GetTryCatchFrameAddr() != nullptr);
+            Assert(tryCatchFrameAddr != nullptr);
```

Comment:
>Assert(tryCatchFrameAddr != nullptr); [](start = 12, length = 37)

remove this assert and add Assert(framePtr == tryCatchFrameAddr); in catch block.